tests/console/velociraptor_client_event_collect.pm

```diff
@@ -87,39 +87,103 @@ sub run {
     # add client monitoring
     my @artifacts = qw(SUSE.Linux.Events.DNS SUSE.Linux.Events.ExecutableFiles SUSE.Linux.Events.ImmutableFile SUSE.Linux.Events.NewFiles SUSE.Linux.Events.NewFilesNoOwner SUSE.Linux.Events.NewHiddenFile SUSE.Linux.Events.NewZeroSizeLogFile SUSE.Linux.Events.Packages SUSE.Linux.Events.ProcessStatuses SUSE.Linux.Events.SSHLogin SUSE.Linux.Events.Services SUSE.Linux.Events.SshAuthorizedKeys SUSE.Linux.Events.SystemLogins SUSE.Linux.Events.TCPConnections SUSE.Linux.Events.Timers SUSE.Linux.Events.UserAccount SUSE.Linux.Events.UserGroupMembershipUpdates);
     foreach my $artifact (@artifacts) {
-        assert_script_run("velociraptor-client --api_config ~/api.config.yaml query 'SELECT add_client_monitoring(artifact=\"$artifact\") FROM scope()'");
+        script_run("velociraptor-client --api_config ~/api.config.yaml query 'SELECT add_client_monitoring(artifact=\"$artifact\") FROM scope()'");
     }
 
```

Comment:
A couple of these artifacts have a default `period` parameter that's too long for testing. You can update by adding a couple of lines to run these queries:

```
SELECT add_client_monitoring(artifact="SUSE.Linux.Events.Packages", parameters=dict(period="10")) FROM scope()
SELECT add_client_monitoring(artifact="SUSE.Linux.Events.SshAuthorizedKeys", parameters=dict(period="10")) FROM scope()
```